README.md

```diff
@@ -76,6 +76,52 @@ steps:
     run: riff run cargo build -- --release
 ```
 
+### Prompt Customization
+
+It is possible to customize your shell's prompt to display when you're in a Riff
```

Comment:
I typically push for more direct/personal. "You can customize..." grabs the reader better.